content/articles/calculating-the-distance-between-two-points-using-geopy/index.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /how-to-build-discord-bot-with-javascript/
+title: How to Build a Discord Bot using JavaScript
+description: This tutorial will help readers understand how to build a Discord bot using JavaScript. The bot will be able to answer certain messages automatically.
+author: kamau-wambui
+date: 2021-12-28T00:00:00-17:33
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/how-to-build-discord-bot-with-javascript/hero.png
+    alt: Build a Discord Bot using JavaScript Hero Image
+---
+Discord is a robust communication app used by a large community of gamers, developers, and cryptocurrency enthusiasts. 
+<!--more-->
+
+
+
+
+Geopy is a Python library that simplifies the process of calculating geographic distances between two points. 
+
+It makes it easier for developers to retrieve coordinates of various points using third-party geocoders, as well as other data sources.
+
+In this article, we'll look at alternative ways of calculating the distance between any two points. 
+
+We will explain how this process is possible using [geopy](https://geopy.readthedocs.io/en/stable/). This is the primary Python library for calculating distance.
+
+You can install `geopy` using the following command:
+
+```Python
+pip install geopy
+```
+
+Using Geopy, we can calculate or find the distance between two positions using the below methods:
+
+- Geodesic measure.
+- The great circle distance.
+- Herversine formula.
+
+### Geodesic measure
```

Comment:
In this section, you appear to be defining geodesic several times thus repeating yourself. Please make changes.